fe/src/main/java/org/apache/doris/plugin/PluginZip.java

```diff
@@ -113,13 +113,13 @@ Path downloadRemoteZip(Path targetPath) throws IOException, UserException {
         cleanPathList.add(zip);
 
         // download zip
-        try (InputStream in = openUrlInputStream(source)) {
+        try (InputStream in = Util.getInputStreamFromUrl(source, null, 5000, 5000)) {
```

Comment:
Should be static const value.
If the timeout isn't a config, I think set a larger value maybe better when the plugin resource is big and network is poor.

Reply:
I change it to a config